packages/supi/test/install/misc.ts

```diff
@@ -1148,3 +1148,24 @@ test('installing a package that has a manifest with byte order mark (BOM)', asyn
 
   await project.has('paralleljs')
 })
+
+test('ignore files in node_modules', async (t: tape.Test) => {
+  const project = prepareEmpty(t)
+  const reporter = sinon.spy()
+
+  await fs.mkdir('node_modules')
+  await fs.writeFile('node_modules/foo', '', 'utf8')
```

Comment:
You should also test that this file exists after the `addDependenciesToPackage(…)` call.